docs/source/tutorial/gpu.rst

```diff
@@ -359,8 +359,7 @@ In order to make a copy, we can use :meth:`Link.copy` method.
 
 .. testcode::
 
-   import copy
-   model_1 = copy.deepcopy(model_0)
+   model_1 = L.copy(model_0)
```

Comment:
`model_1 = model_0.copy()`

(`L` is the module `chainer.links`, not the class `chainer.Link`.)